install

```diff
@@ -351,8 +352,8 @@ fi
 echo
 for shell in $shells; do
   [[ "$shell" = fish ]] && continue
-  [ $shell = zsh ] && dest=${ZDOTDIR:-~}/.zshrc || dest=~/.bashrc
-  append_line $update_config "[ -f ~/.fzf.${shell} ] && source ~/.fzf.${shell}" "$dest" "~/.fzf.${shell}"
+  [ "$shell" = zsh ] && dest=${ZDOTDIR:-~}/.zshrc || dest=~/.bashrc
+  append_line $update_config "[ -f $HOME/fzf.${shell} ] && source $HOME/fzf.${shell}" "$dest" "$HOME/fzf.${shell}"
```

Comment:
We want to add literal `~` to the file (e.g. `[ -f ~/.fzf.bash ] && source ~/.fzf.bash`), so this was intentional.

Reply:
I see I dropped the `.` in `.fzf` my mistake too. I'll push an update today.